pageserver/src/http/routes.rs

```diff
@@ -867,6 +884,12 @@ async fn tenant_create_handler(
     let tenant_conf =
         TenantConfOpt::try_from(&request_data.config).map_err(ApiError::BadRequest)?;
 
+    // TODO: make generation mandatory here once control plane supports it.
+    let generation = request_data
+        .generation
+        .map(Generation::new)
+        .unwrap_or(Generation::none());
```

Comment:
The create path does accepting the generation from the request, or uses Generation::none if the request omits the generation -- this is the same as attach, just without the extra logic that returns 400 if generation is omitted and control_plane_api is set.

In 891f7c4ccc78505f9d7ea255ca30787e62f68852 I've added a helper that makes it more obvious that they're doing the same thing, and also added the generation to load (that replaces the previous comment that /load was unsafe... we should just make it safe).  I still see /load going away soon, but generation-enabling it will let Gleb do the generations support without having to simultaneously change code that uses load (other than augmenting it with a generation).

Reply:
I like it, resolving this.